tests/unit/instance-initializers/axe-component-test.js

```diff
@@ -40,12 +40,14 @@ test('initializer should not re-open Ember.Component more than once', function(a
 test('audit is run on didRender when not in testing mode', function(assert) {
   initialize(application);
 
+  // In order for the audit to run, Ember.testing has to be false
+  // when the component is initialized
+  Ember.testing = false;
   let component = Ember.Component.create({});
+  Ember.testing = true;
```

Comment:
That of course makes tests like this a touch harder to write, but oh well–everything else is much faster.